crates/stackable-operator/src/cli.rs

```diff
@@ -245,6 +246,17 @@ pub struct ProductOperatorRun {
     /// Provides a specific namespace to watch (instead of watching all namespaces)
     #[arg(long, env, default_value = "")]
     pub watch_namespace: WatchNamespace,
+
+    /// Don't maintain the CustomResourceDefinitions (CRDs) the operator is responsible for.
+    ///
+    /// Maintenance includes creating the CRD initially, adding new versions and keeping the TLS
+    /// certificate of webhooks up to date. Turning this off can be desirable to reduce the RBAC
+    /// permission of the operators.
+    ///
+    /// WARNING: If you disable CRD maintenance you are responsible for maintaining it, including
+    /// the points above.
```

Comment:
nit: I like the overall comment, nicely done. Just some small nits:


```suggestion
    /// Don't maintain the CustomResourceDefinitions (CRDs) the operator is responsible for.
    ///
    /// Maintenance includes creating the CRD initially, adding new versions and keeping the TLS
    /// certificate of webhooks up to date. Turning this off can be desirable to reduce the RBAC
    /// permissions of the operator.
    ///
    /// WARNING: If you disable CRD maintenance you are responsible for maintaining it, including,
    /// but not limited to, the points above.
```